index.js

```diff
@@ -4,15 +4,37 @@ console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+header.style.color = '#088F8F'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+const profile = document.querySelector('div#profile')
+const travelerImg = profile.querySelector('img')
 
+travelerImg.src = traveler.photo
+travelerImg.alt = traveler.name
+
+profile.querySelector('h2').innerHTML = traveler.name
+profile.querySelector('em').innerHTML = traveler.nickname
```

Comment:
Nice organization and good thinking scoping your querySelector search to the profile (:

Reply:
Always use `textContent` over `innerHTML` where possible, especially if you know you're setting plain text. See here to read about the security implications of `innerHTML`: https://developer.mozilla.org/en-US/docs/Web/API/Element/innerHTML#security_considerations

```suggestion
console.log('This is what the traveler object looks like: ', traveler)
const profile = document.querySelector('div#profile')
const travelerImg = profile.querySelector('img')

travelerImg.src = traveler.photo
travelerImg.alt = traveler.name

profile.querySelector('h2').textContent = traveler.name
profile.querySelector('em').textContent = traveler.nickname
```